web/src/beta/components/TabMenu/index.tsx

```diff
@@ -18,11 +18,18 @@ export type TabObject = {
 export type Props = {
   tabs: TabObject[];
   selectedTab: string;
+  isScroll?: boolean;
```

Comment:
Could you replace all `isScroll` with `scrollable`? I think `isScroll` is a little bit confusing.